appveyor.yml

```diff
@@ -2,21 +2,28 @@ version: 1.0.{build}
 branches:
   only:
   - windows_port
+clone_depth: 5
```

Comment:
That may be an acceptable compromise.

BTW: I just made a `git clone --depth=10 --branch=windows_port` but I get more than 10 commits on the first parent history line... I don't get how that works.